hw15/car.py

```diff
@@ -0,0 +1,9 @@
+class Car:
+    PRICE = 40
```

Comment:
would be better to move this class attribute on instance level.
just for example: if I create two cars sell these cars and after that change this attribute - cars that I sold out also have this price

Reply:
unless it's a condition. I cant see description ot task here..